onboarding-tutorials/electrical-onboarding/index.rst

```diff
@@ -1,8 +1,47 @@
 Electrical Onboarding
-=====================
+#####################
 
 .. toctree::
    :maxdepth: 2
    :caption: Tutorial Pages
 
-   mplab-tutorial
+   mplab-tutorial.rst
+
+Welcome new Members
+*******************
+Please complete required safety training, and install the following required software on your computer.
+
+Required Softwares
+******************
+
+git
+===
+Git is a version control system, install it from `git website <https://git-scm.com/downloads>`_  or your package manager
+
+KiCAD
+=====
+KiCAD is the PCB design software. We use KiCAD version 8, install KiCAD from `KiCAD website <https://www.kicad.org/download/>`_ or your package manager
+
+MPLab and XC compilers
+======================
+MPLab is an IDE(Integrated Development Environment) for Microchip PIC microcontrollers, XC8 and XC16 are the compilers. Use the following links to download them, `MPLAB X IDE <https://www.microchip.com/en-us/tools-resources/develop/mplab-x-ide>`_ and the `XC8 and XC16 compilers <https://www.microchip.com/en-us/tools-resources/develop/mplab-xc-compilers>`_
```

Comment:
space between IDE and (integrated dev enviro)

Reply:
I think this is fine